flow.yml

```diff
@@ -36,7 +43,7 @@ executors:
       '/': rank
     tls: true
     external: true
-    needs: [dalle, diffusion]
+    needs: [stable, dalle, diffusion]
```

Comment:
this does not make sense if u do not make stable need of gateway

Reply:
Yeah, I wondered about this. I will remove it, thanks.

Reply:
Actually, I get the error:
```
WARNI… Flow@77837 ['stable'] are "floating" in this flow     [08/19/22 14:43:56]
       with no deployment receiving from them, you may want                     
       to double check if it is intentional or some mistake
```

It does seem like it uses this gateway (`rerank`) to rank candidates returned from stable. I will leave it in place.